addenda.go

```diff
@@ -1,10 +1,15 @@
 package ach
 
 import (
+	"flag"
 	"fmt"
 	"strings"
 )
 
+func init() {
+	flag.Lookup("alsologtostderr").Value.Set("true")
```

Comment:
I would like to keep this as a library and return the error to your caller. If you choose to handle the error by logging it, by definition it’s not an error anymore — you handled it. I assume we can remove this and you can log the error in your calling code rather than adding it to the ACH library?

Reply:
Yes this can be pulled out -- it is a remnant of glog while working on this